code/game/gamemodes/cult/cult_items.dm

```diff
@@ -45,9 +45,9 @@
 /obj/item/weapon/melee/cultblade/dagger/afterattack(mob/living/target as mob, mob/living/carbon/human/user as mob)
 	..()
 	var/mob/living/carbon/human/bleeder = target
-	if(!(cooldown > world.time) && (!bleeder.stat || !(bleeder.species.flags & NO_BLOOD)))
-		user.visible_message("<span class='warning'>The runes on the blade absorb the blood of [target]!</span>")
-		bleeder.drip(500)
+	if(!(cooldown > world.time) && ((bleeder.stat != DEAD) || !(bleeder.species.flags & NO_BLOOD)))

```

Comment:
That `||` should be an `&&` or it's going to instakill any species that has blood (which is most of them)